packages/server/lib/cloud/protocol.ts

```diff
@@ -170,7 +173,7 @@ export class ProtocolManager implements ProtocolManagerShape {
     this.invokeSync('resetTest', testId)
   }
 
-  async uploadCaptureArtifact (uploadUrl: string) {
+  async uploadCaptureArtifact ({ uploadUrl, timeout }) {
```

Comment:
is this a blocking action before the next spec starts?

@mjhenkes do we measure how low this upload typically takes?

Reply:
Today yes it is. This is one of the things I want to evaluate with: https://github.com/orgs/cypress-io/projects/10/views/22?pane=issue&itemId=30422529